INSTALL.md

```diff
@@ -52,7 +53,7 @@ Please follow each part of this documentation in order to run your own instance
 
 ## .env settings
 
-Saskatoon uses `python-dotenv` to manage local environment settings. Copy the [saskatoon/env.template](saskatoon/env.template) file into `saskatoon/.env` and adapt it to your needs. 
+Saskatoon uses `python-dotenv` to manage local environment settings. Copy the [saskatoon/env.template](saskatoon/env.template) file into `saskatoon/.env` and adapt it to your needs.
```

Comment:
@mina-andajos can we change this line for `django-dotenv` as discussed in #137 ?

Also would it be okay to rebase these changes onto develop?

Reply:
will do 👍

Reply:
changed it and rebased 👍